.github/workflows/pytest.yml

```diff
@@ -9,12 +9,16 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
Why support multiple versions of python? This seems like extra work.

Reply:
The default version of python for the Raspberry Pi is currently 3.9, thought it would be best to support the default version and updates afterwards until the Raspberry Pi makes a newer version its default.